contracts/mainnet/connectors/compound/v3-rewards/main.sol

```diff
@@ -17,55 +17,48 @@ abstract contract CompoundV3RewardsResolver is Events, Helpers {
 	 * @dev Claim rewards and interests accrued in supplied/borrowed base asset.
 	 * @notice Claim rewards and interests accrued.
 	 * @param market The address of the market.
-	 * @param account The account of which the rewards are to be claimed.
-	 * @param accrue Should accrue the rewards and interest before claiming.
 	 * @param setId ID stores the amount of rewards claimed.
 	 */
 	function claimRewards(
 		address market,
-		address account,
-		bool accrue,
 		uint256 setId
 	) public returns (string memory eventName_, bytes memory eventParam_) {
-		uint256 rewardsOwed = cometRewards.getRewardOwed(market, account).owed;
-		cometRewards.claim(market, account, accrue);
+		uint256 rewardsOwed = cometRewards.getRewardOwed(market, address(this)).owed;
+		cometRewards.claim(market, address(this), true);
 
 		setUint(setId, rewardsOwed);
 
-		eventName_ = "LogRewardsClaimed(address,address,uint256,uint256,bool)";
-		eventParam_ = abi.encode(market, account, rewardsOwed, setId, accrue);
+		eventName_ = "LogRewardsClaimed(address,address,uint256,uint256)";
+		eventParam_ = abi.encode(market, address(this), rewardsOwed, setId);
 	}
 
 	/**
 	 * @dev Claim rewards and interests accrued in supplied/borrowed base asset.
 	 * @notice Claim rewards and interests accrued and transfer to dest address.
 	 * @param market The address of the market.
-	 * @param account The account of which the rewards are to be claimed.
-	 * @param dest The account where to transfer the claimed rewards.
-	 * @param accrue Should accrue the rewards and interest before claiming.
+	 * @param owner The account of which the rewards are to be claimed.
+	 * @param manager The account where to transfer the claimed rewards.
 	 * @param setId ID stores the amount of rewards claimed.
 	 */
-	function claimRewardsTo(
+	function claimRewardsOnBehalfOf(
 		address market,
-		address account,
-		address dest,
-		bool accrue,
+		address owner,
+		address manager,
```

Comment:
@Richa-iitr I think we can change it back `to`.
```suggestion
		address to,
```

Reply:
done